tests/test-class-apple-news.php

```diff
@@ -0,0 +1,53 @@
+<?php
+
+use \Apple_News as Apple_News;
+use \Apple_Exporter\Settings as Settings;
+
+class Apple_News_Test extends WP_UnitTestCase {
+
+	public function setup() {
+		parent::setup();
+		$this->settings = new Settings();
+	}
+
+	public function testGetFilename() {
+		$url = 'http://someurl.com/image.jpg?w=150&h=150';
+		$filename = Apple_News::get_filename( $url );
+
+		$this->assertEquals( 'image.jpg', $filename );
+	}
+
+	public function testVersion() {
+		$plugin_data = apple_news_get_plugin_data();
+		$this->assertEquals( $plugin_data['Version'], Apple_News::$version );
+	}
+
+	public function testMigrateSettings() {
+		update_option( 'use_remote_images', 'yes' );
```

Comment:
Do you need to revert this option back to its original value after running this test?  Maybe this is handled somewhere else, or maybe it's not necessary, but just thought I'd ask.